webapp/channels/src/components/analytics/true_up_review.tsx

```diff
@@ -221,10 +220,6 @@ const TrueUpReview: React.FC = () => {
         return null;
     }
 
-    if (telemetryEnabled) {
-        return null;
-    }
```

Comment:
If we remove this condition then this will show for all the ones that do have telemetry enabled and we already have their true-up information. I think that's the reason this was originally added. Should we instead add a condition like: 

```
if (telemetryEnabled && !airgapped) {
        return null;
    }
```

Reply:
Hey Maria, heads up on some context: https://hub.mattermost.com/private-core/pl/4q14jqcnatf7jj3mhcomh7cgny

Basically, we want to show the button even with telemetry enabled to align with SalesOps process

Reply:
Ah gotcha, then removing the check as-is will work as intended 👍